jvm/controls/src/main/resources/css/add-plugin-dialog.css

```diff
@@ -32,12 +31,12 @@
 .add-plugin-dialog__title {
     -fx-font-size: 28px;
     -fx-font-weight: bold;
-    -fx-text-fill: -wa-primary;
+    -fx-text-fill: -wa-highlight-text;
```

Comment:
I wouldn't change the name of primary, secondary, etc... colors. You can have different colors for primary in every theme